src/MarkdownEditor.js

```diff
@@ -65,7 +78,7 @@ var MarkdownEditor = React.createClass({
     };
 
     if(this.props.styles.styleMarkdownEditorHeader) {
-      styleMarkdownEditorHeader = this.props.styles.styleMarkdownEditorHeader;
+      Object.assign(styleMarkdownEditorHeader, this.props.styles.styleMarkdownEditorHeader);
```

Comment:
Is it backward compatible? I thought Object.assign was a es6 feature only, meaning we will probably need a polyfill of some sort. Wdyt?